.github/workflows/closed.yml

```diff
@@ -0,0 +1,67 @@
+name: Remove PR Previews
+
+on:
+  pull_request:
+    types: [closed]
+
+concurrency: preview-teardown-${{ github.ref }}
+
+jobs:
+  list-environments:
+    runs-on: ubuntu-latest
+    outputs:
+      hasNextJsEnv: ${{ steps.list-environments.outputs.hasNextJsEnv }}
+      hasVanillaJsEnv: ${{ steps.list-environments.outputs.hasVanillaJsEnv }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: List repo environments
+        id: list-environments
+        uses: actions/github-script@v6
+        env:
+          GITHUB_TOKEN: ${{ secrets.PIE_BOT_TOKEN }}
+        with:
+          script: |
+            const pull_number = context.issue.number;
```

Comment:
To keep the camelCase standard, can it be `pullNumber` or `pullRequestNumber` instead?